sdk/storage/src/datalake/path_client.cpp

```diff
@@ -47,6 +46,7 @@ namespace Azure { namespace Storage { namespace DataLake {
         {
           length = std::stoll(GetSubstringTillDelimiter('\n', subRangeString, cur)) - offset + 1;
         }
+        // else raise exception?
```

Comment:
Yes, please throw an exception if parse failed

Reply:
I'll throw an exception on this.